Updates to install promo content for PWAs (#3235)
GoogleChrome/web.dev: src/site/content/en/progressive-web-apps/promote-install/index.md

  completed a purchase.
* Combine more than one of these techniques in different parts of your PWA,
  but be careful not to overwhelm or annoy your user with install promotion.
* Only show the promotion **after** the
  [`beforeinstallprompt` event](/customize-install/#beforeinstallprompt) has
  been fired.

## Automatic browser promotion {: #browser-promotion }

When [certain criteria](/install-criteria/) are met, most browsers will
automatically indicate to the user that your Progressive Web App is
installable. For example, desktop Chrome shows an install button in the omnibox.

<div class="w-columns">
  <figure class="w-figure" id="browser-install-promo">
    <img src="how-does-it-work_desktop.png"
        alt="Screenshot of omnibox with install indicator visible.">
    <figcaption class="w-figcaption">
      Browser provided install promotion (desktop)
    </figcaption>
  </figure>
  <figure class="w-figure">
    <img src="how-does-it-work_mobile.png"
        alt="Screenshot of browser provided install promotion.">
    <figcaption class="w-figcaption">
      Browser provided install promotion (mobile)
    </figcaption>
  </figure>

> </div>

Reviewer: @robdodson these images have also been used in another post that @petele is updating. Pete is currently duplicating the images in the `/install-criteria` directory. Would you prefer to use a single source of the images?

Reviewer 2: I think I'm ok with either approach.

- If we update the image in /install-criteria then this post will receive the update (yay!)
- However, if we delete the image in /install-criteria then we have to remember to delete it here (boo)
- If we duplicate the images then we have to remember to update them in both places but it does mean that removing post A will not have any negative effect on post B.

Reviewer: I was also thinking that it was fine to duplicate because of the risk that we update one article and forget that the other also depends on it. I just wanted to make sure I wasn't missing anything from an engineering perspective.

<div class="w-clearfix">&nbsp;</div>

### Simple install button {: #simple-button }

The simplest possible UX is to include an 'Install' or 'Get app' button at an
appropriate location in your web content. Ensure the button doesn't block
other important functionality and is out of the way of the user's flow in

> the user's journey for your application.

Reviewer: @petele does this text have some extra words, or is this written as intended: `out of the way of the user's flow in the user's journey`

Reviewer 3: I feel this para might do with a re-write, it feels a little clunky. Basically what we are saying is a simple install button is an easy way to engagement users just make sure it is placed in the UI where is can be seen but doesnt block the user from doing anything else, for example on a shopping site is doesnt block a user from adding something to their cart or reading an article on a news site etc.

Reviewer: ```suggestion
other important functionality and is out of the way of the user's journey
through your application.
```
I think this is good enough

appears on  page load, or out of context, it can be easily missed, or lead
to cognitive overload. When this happens, users will simply dismiss everything
they see. And remember, new users to your site may not be ready to install your
PWA. Therefore, it's best to wait until you have strong interest signals from
the user before using this pattern, for example, repeat visits, a user sign
in, or similar conversion event.

<figure class="w-figure w-figure--inline-right">
  <img src="temporary-ui_mobile.png" alt="Custom install banner as snackbar.">
  <figcaption class="w-figcaption">
    A dismissible snackbar indicating the PWA is installable.
  </figcaption>
</figure>

Make sure you:

* Show the snackbar for between 4 and 7 seconds to give users enough
  time to see and react to it, and without getting in the way.
* Avoid showing it over other temporary UI such as banners, etc.
* Wait until you have strong interest signals from the user before
  using this pattern, for example, repeat visits, a user sign in,
  or similar conversion event.

<div class="w-clearfix">&nbsp;</div>

## After conversion

Immediately after a user conversion event, for example after a purchase on

> an e-commerce site is an excellent opportunity to promote the installation of

Reviewer: ```suggestion
an e-commerce site, is an excellent opportunity to promote the installation of
```
